bluesky_kafka/logging_utils.py

```diff
@@ -0,0 +1,129 @@
+# See https://docs.confluent.io/platform/current/installation/configuration/producer-configs.html and
+#     https://docs.confluent.io/platform/current/installation/configuration/consumer-configs.html
+CONFIG_KEY_WHITELIST = {
+    ##
+    # -- Common Keys
+    "bootstrap.servers",
+    "client.dns.lookup",
+    "client.id",
+    "connections.max.idle.ms",
+    "receive.buffer.bytes",
+    "request.timeout.ms",
+    # "sasl.client.callback.handler.class",
+    # "sasl.jaas.config",
+    # "sasl.kerberos.service.name",
+    # "sasl.login.callback.handler.class",
+    # "sasl.login.class",
+    # "sasl.mechanism",
+    # "sasl.oauthbearer.jwks.endpoint.url",
+    # "sasl.oauthbearer.token.endpoint.url",
```

Comment:
I don't usually like having a heap of commented out code like this, but it is handy to see at a glance what keys are getting redacted so I left it in for now.